.travis.yml

```diff
@@ -105,10 +105,10 @@ install:
 
 script:
   - if [[ $TEST_TARGET == 'default' ]]; then
-      python -m iris.tests.runner --default-tests --system-tests --print-failed-images;
+      python -m iris.tests.runner --default-tests --system-tests;
```

Comment:
I think it could make sense to remove the support for the  "print-failed-images" key from setup.py.